src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.dao.CarDaoImpl;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer bmwManufactured = new Manufacturer();
+        bmwManufactured.setCountry("Germany");
+        bmwManufactured.setName("BMWManufactured");
+        manufacturerDao.create(bmwManufactured);
+
+        Car bmw = new Car();
+        bmw.setModel("bmw");
+        bmw.setManufacturer(bmwManufactured);
+
+        DriverDao driverDao = new DriverDaoImpl();
+
+        List<Driver> bmvDrivers = new ArrayList<>();
+        bmvDrivers.add(driverDao.get(1L).get());
+        bmvDrivers.add(driverDao.get(2L).get());
+        bmvDrivers.add(driverDao.get(3L).get());
+        bmw.setDrivers(bmvDrivers);
+
+        CarDao newCar = new CarDaoImpl();
```

Comment:
use meaningful variable names!
```suggestion
        CarDao carDao = new CarDaoImpl();
```